libs/hdf-converters/data/converters/cciListXml2json.ts

```diff
@@ -2,28 +2,31 @@ import fs from 'fs';
 import * as _ from 'lodash';
 import xml2js from 'xml2js';
 
+// Documentation is located at https://github.com/mitre/heimdall2/wiki/Control-Correlation-Identifier-(CCI)-Converter.
 const parser = new xml2js.Parser();
 const pathToInfile = process.argv[2];
-const pathToOutfile = process.argv[3];
+const pathToCci2NistOutfile = process.argv[3];
```

Comment:
why are we creating two separate files if our workflow will just combine them?

let's move what you're doing in the workflow to this file as well.  consequently we still only need the one file.

edit: actually thinking about it a bit more, I think what I want to have happen is that this script does make two separate files and then these raw json objects are imported by the module and wrapped by the object that exposes them to the outside.  you can sidestep the prettier stuff doing this as well by using the additional options in json.stringify for the json blobs and the single file that exposes them will no longer need to be touched at all.  it also makes it so we don't need to do the 'append a whole bunch of stuff together' thing you were doing too.